test/onnx/test_fx_op_consistency.py

```diff
@@ -57,11 +57,11 @@
         "acosh",
         "add",
         "addmm",
-        "all",
+        # "all",    - onnxscript doesn't handle aten::all.dims
         "allclose",
         "amax",
         "amin",
-        "any",
+        # "any",    - onnxscript doesn't handle aten::any.dims
```

Comment:
Apparently this is testing out-of-tree code, so can't be fixed in this PR
https://github.com/microsoft/onnxscript/blob/f8046e17490a3222d4ee4bca346b4449a2d30ae4/onnxscript/function_libs/torch_lib/ops/core.py#L404

Reply:
cc @justinchuby

Reply:
The current best thing to do is to xfail it here: https://github.com/pytorch/pytorch/pull/110310/files/73dd8600a8097f98b675510024e01ea1a3d6af5d#diff-db2f78a51511bb172cbfde1b2f68272b8b33049abe2571cded27bcd0f3ae5fa4R176 and track it with an issue. 

I will discuss with the team to find a better solution forward. Thanks!